service/src/main/resources/application-control-plane.yml

```
@@ -11,14 +11,14 @@ twds:
     batch-write-record-sink: "rawls"
     succeed-on-completion: false
     rawls-bucket-name: ${SERVICE_GOOGLE_BUCKET}
+    permission-sync: true
```

Since this is specific to one type of import (TDR snapshots) and allows a user to sync permissions, suggest a more specific name:

```suggestion
    enable-tdr-permission-sync: true
```

---

renamed